nixos/common/networking.nix

```diff
@@ -24,4 +24,9 @@
   systemd.services.systemd-networkd.stopIfChanged = false;
   # Services that are only restarted might be not able to resolve when resolved is stopped before
   systemd.services.systemd-resolved.stopIfChanged = false;
+
+  # Allows to find machines on the local network by name
+  systemd.network.networks."99-ethernet-default-dhcp".networkConfig.MulticastDNS = lib.mkDefault "yes";
+  systemd.network.networks."99-wireless-client-dhcp".networkConfig.MulticastDNS = lib.mkDefault "yes";
+  networking.firewall.allowedUDPPorts = [ 5353 ]; # Multicast DNS
```

Comment:
I also learned today that without this rule, you cannot resolve other mdns names as well. I.e. many printers these days only use mdns for discovery, which works on Ubuntu but breaks on NixOS.

Reply:
Hence avahi opens it by default because otherwise it is quite literal useless but pipe wire only works with avahi, so I kinda need to stick to it.

Reply:
Would avahi create a conflict with these rules? We can also check for avahi here.